test-data/unit/check-tuples.test

```diff
@@ -1421,3 +1421,10 @@ t6: Tuple[int, int, int, int, int, int, int, int, int, int, int, int] = (1, 2, 3
                                                                     # E: Incompatible types in assignment (expression has type Tuple[int, int, ... <15 more items>], variable has type Tuple[int, int, ... <10 more items>])
 
 [builtins fixtures/tuple.pyi]
+
+[case testTupleWithStarExpr]
+from typing import Tuple
+points = (1, 2)  # type: Tuple[int, int]
```

Comment:
Instead of using all `int` types, use different types so that each of `x`, `y` and `z` will get distinct types. This way we can test that the types are propagated in the correct order.